app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt

```diff
@@ -64,16 +52,15 @@ class NotificationScheduler(
     }
 
     private suspend fun scheduleUseOurAppNotification() {
-        if (userStageStore.useOurAppNotification()) {
+        if (variant().hasFeature(VariantManager.VariantFeature.InAppUsage) && addToHomeCapabilityDetector.isAddToHomeSupported() && useOurAppNotification.canShow()) {
```

Comment:
any reason `addToHomeCapabilityDetector.isAddToHomeSupported()` is not part of the logic of `useOurAppNotification.canShow()`?

Reply:
Nothing specific, I've moved that to the `canShow()` fun.